src/xcode/ENA/ENAUITests/Home/ENAUITestsHome.swift

```diff
@@ -84,4 +84,82 @@ class ENAUITests_01_Home: XCTestCase {
+		snapshot("homescreenrisk_level_\(riskLevel)_\(String(format: "%04d", (screenshotCounter.inc() )))")
+		app.swipeUp()
+		snapshot("homescreenrisk_level_\(riskLevel)_\(String(format: "%04d", (screenshotCounter.inc() )))")
+	}
+	
+	func test_screenshot_homescreen_riskCardLow() throws {
+		var screenshotCounter = 0
+		let riskLevel = "low"
+		app.setPreferredContentSizeCategory(accessibililty: .accessibility, size: .XS)
+		app.launchArguments.append(contentsOf: ["-riskLevel", riskLevel])
+		app.launch()
+
+		// only run if home screen is present
+		XCTAssert(app.buttons["AppStrings.Home.rightBarButtonDescription"].waitForExistence(timeout: 5.0))
+		snapshot("homescreenrisk_level_\(riskLevel)_\(String(format: "%04d", (screenshotCounter.inc() )))")
+		app.swipeUp()
+		snapshot("homescreenrisk_level_\(riskLevel)_\(String(format: "%04d", (screenshotCounter.inc() )))")
+	}
+	
+	func test_screenshot_homescreen_riskCardInactive() throws {
+		var screenshotCounter = 0
+		let riskLevel = "inactive"
+		app.setPreferredContentSizeCategory(accessibililty: .accessibility, size: .XS)
+		app.launchArguments.append(contentsOf: ["-riskLevel", riskLevel])
+		app.launch()
+
+		// only run if home screen is present
+		XCTAssert(app.buttons["AppStrings.Home.rightBarButtonDescription"].waitForExistence(timeout: 5.0))
+		snapshot("homescreenrisk_level_\(riskLevel)_\(String(format: "%04d", (screenshotCounter.inc() )))")
+		app.swipeUp()
+		snapshot("homescreenrisk_level_\(riskLevel)_\(String(format: "%04d", (screenshotCounter.inc() )))")
+
+	}
+	
+	func test_screenshot_homescreen_riskCardUnknownUnknownInitial() throws {
+		var screenshotCounter = 0
+		let riskLevel = "unknownInitial"
+		app.setPreferredContentSizeCategory(accessibililty: .accessibility, size: .XS)
+		app.launchArguments.append(contentsOf: ["-riskLevel", riskLevel])
+		app.launch()
+
+		// only run if home screen is present
+		XCTAssert(app.buttons["AppStrings.Home.rightBarButtonDescription"].waitForExistence(timeout: 5.0))
+		snapshot("homescreenrisk_level_\(riskLevel)_\(String(format: "%04d", (screenshotCounter.inc() )))")
+		app.swipeUp()
+		snapshot("homescreenrisk_level_\(riskLevel)_\(String(format: "%04d", (screenshotCounter.inc() )))")
+	}
+	
+	func test_screenshot_homescreen_riskCardUnknownUnknownOutdated() throws {
+		var screenshotCounter = 0
+		let riskLevel = "unknownOutdated"
+		app.setPreferredContentSizeCategory(accessibililty: .accessibility, size: .XS)
+		app.launchArguments.append(contentsOf: ["-riskLevel", riskLevel])
+		app.launch()
+
+		// only run if home screen is present
+		XCTAssert(app.buttons["AppStrings.Home.rightBarButtonDescription"].waitForExistence(timeout: 5.0))
+		snapshot("homescreenrisk_level_\(riskLevel)_\(String(format: "%04d", (screenshotCounter.inc() )))")
+		app.swipeUp()
+		snapshot("homescreenrisk_level_\(riskLevel)_\(String(format: "%04d", (screenshotCounter.inc() )))")
+	}
+}
+
+extension Int {
```

Comment:
Please move this into its own extension file. Since it seems to be needed from more then just this UITest File.

Reply:
Thanks. Refactored it into own file.